wsproto/connection.py

```diff
@@ -195,27 +176,35 @@ def closed(self):
 
     def bytes_to_send(self, amount=None):
         """
-        Return any data that is to be sent to the remote peer.
+        Returns some data for sending out of the internal data buffer.
+
+        This method is analogous to ``read`` on a file-like object, but it
+        doesn't block. Instead, it returns as much data as the user asks for,
+        or less if that much data is not available. It does not perform any
+        I/O, and so uses a different name.
 
-        :param amount: (optional) The maximum number of bytes to be provided.
-            If ``None`` or not provided it will return all available bytes.
+        :param amount: (optional) The maximum amount of data to return. If not
+            set, or set to ``None``, will return as much data as possible.
         :type amount: ``int``
+        :returns: A bytestring containing the data to send on the wire.
+        :rtype: ``bytes``
         """
-
         if amount is None:
             data = self._outgoing
             self._outgoing = b''
         else:
             data = self._outgoing[:amount]
             self._outgoing = self._outgoing[amount:]
-
         return data
 
     def receive_bytes(self, data):
         """
-        Pass some received bytes to the connection for processing.
+        Pass some received data to the connection for handling.
+
+        A list of events that the remote peer triggered by sending this data can
+        be retrieved with :meth:`events <wsproto.connection.WSConnection.events>`.
```

Comment:
Another good place to use sphinx's magic `~`

Reply:
👍